apps/ledger-live-desktop/tests/page/settings.page.ts

```diff
@@ -57,12 +58,18 @@ export class SettingsPage extends AppPage {
     await this.experimentalDevModeToggle.click();
   }
 
-  async changeCounterValue() {
+  @step("Change counter value")
+  async changeCounterValue(currency: string) {
```

Comment:
might make sense to pass a `currency` and to store `EUR` in `currency` enum, wdyt?

Reply:
I don't know since we are only using EUR. Also, for now in currency I only have Crypto Currency such as BTC, ETH... so idk if it's a good idea to mix fiat and crypto